### PR TITLE
more cleanup

### PR DESCRIPTION
**Pull request by jumanjiman**
* webserver: send 404 if client requests non-existent endpoint
* webserver: walk the flags just once
* webserver: consolidate log/print statements into log method
* pep8: add and fix check for 501 Line too long
* pep8 is now pycodestyle
* test fixture: use (shorter) absolute path to report errors
* replace pycodestyle with flake8, which includes pycodestyle

### CODE REVIEW

File: webserver/server.py [hunk 1]
```diff
@@ -60,75 +63,71 @@ def play(self):
...
-        print (command)
-        print path
+        self.log("program output: " + out)
+        self.log(command)
+        self.log(path)
 
         # test pass string output from proc
         #
-        self.send_response = (200, out)
+        self.send_response(200, out)
```
Comment (jumanjiman) [inline webserver/server.py]: @sometheycallme pylint spotted this one

**Comment by sometheycallme** [comment]
>pylint spotted this one

nice @jumanjiman and awesome commits on this too.   I'm going to spawn the pipeline and run some tests against this.

**Comment by kropped** [comment]
:bell: Staged `jumanjiman_more_cleanup` at revision 39f9a6c299811d52fe386d08e8cae29fde4988dd on 79e0d45c398a

**Comment by kropped** [comment]
:bell: Staged `jumanjiman_more_cleanup` at revision 39f9a6c299811d52fe386d08e8cae29fde4988dd on 9051d12665b7

**Comment by sometheycallme** [comment]
@jumanjiman 

SWEET! the icing was picking up on the ```=```

```self.send_response(200, out)``` is now sending feedback from the running commands.

Check out the response from the controller on the client side connection:

```shell
core@pipeline ~/ansible-security $ curl -v -X POST -d '{"branch_name": "more_cleanup", "git_handle": "jumanjiman", "flags": [{"flag": "
-i", "argument": "hosts"}], "playbook": "fixtures/etc/ansible/play_test.yml"}' http://172.17.0.6:8080/play
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying 172.17.0.6...
* Connected to 172.17.0.6 (172.17.0.6) port 8080 (#0)
> POST /play HTTP/1.1
> Host: 172.17.0.6:8080
> User-Agent: curl/7.47.1
> Accept: */*
> Content-Length: 157
> Content-Type: application/x-www-form-urlencoded
> 
* upload completely sent off: 157 out of 157 bytes
* HTTP 1.0, assume close after body
< HTTP/1.0 200 
< PLAY [local] *******************************************************************
< 
TASK [setup] *******************************************************************
ok: [localhost]

TASK [Ansible Ping Localhost] **************************************************
changed: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

Server: BaseHTTP/0.3 Python/2.7.12
Date: Mon, 11 Jul 2016 02:44:08 GMT
* Closing connection 0
core@pipeline ~/ansible-security $ 
```

**Comment by sometheycallme** [comment]
🚢

**Comment by jumanjiman** [comment]
> self.send_response(200, out) is now sending feedback from the running commands.

👍

**Pull request merged**